vignettes/cpp11_faq.Rmd

```diff
@@ -170,9 +169,9 @@ cpp11::writable::logicals fn2() {
 }
 ```
     
-  * Otherwise this will return a 0 length logical vector:
```

Comment:
Thanks for updating this, it was a recent change in behavior, but I think we have better behavior now!